src/ophys_etl/modules/video/utils.py

```diff
@@ -232,45 +249,57 @@ def create_side_by_side_video(
         create_downsampled_video_h5(
             video_1_path, input_hz,
             tmp_1_h5, output_hz,
-            kernel_size,
+            spatial_filter,
             n_processors)
 
         (min_1,
          max_1) = _min_max_from_h5(tmp_1_h5, quantiles)
 
         logger.info(f'wrote {video_1_path} to {tmp_1_h5}')
 
-        video_array = np.zeros((video_0_shape[0],
-                                video_0_shape[1],
-                                gap+2*video_0_shape[2],
-                                3), dtype=np.uint8)
-
         video_0_uint = _video_array_from_h5(tmp_0_h5,
-                                            min_0,
-                                            max_0,
-                                            reticle)
+                                            min_val=min_0,
+                                            max_val=max_0,
+                                            reticle=reticle,
+                                            video_dtype=video_dtype)
+
+        video_array = np.zeros((video_0_uint.shape[0],
```

Comment:
Why do you declare this array here and again below?

Reply:
Because I was moving things around like a mad man and lost track of where I was...